app/models/user.rb

```diff
@@ -37,7 +37,10 @@ class User < ActiveRecord::Base
   validates :uid, :presence => :true
   validates :provider, :presence => :true
 
+  after_save :notify_facebook_friends
+
   def self.for_provider!(provider, uid, params={})
+    @access_token = params[:provider_token]
```

Comment:
We want to send a notification to users when one of their FB friends joins Givdo.